src/main/java/com/chnu/repository/IProfileRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.chnu.repository;
+
+import com.chnu.model.Profile;
+
+import java.util.Optional;
+
+public interface IProfileRepository extends BaseRepository<Profile,Long> {
+    Optional<Profile> findByProfile(Long userId);
```

Comment:
This method isn`t needful. You can use method from AbstractRepository instead of that.